the-collab-lab/tcl-41-smart-shopping-list
# Jc eb new item form

## Description

Adds new item form to the new item view. Form submits data to the database. List view will display items in the database filtered by user token.

## Related Issue

Closes #4 

## Acceptance Criteria


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |


## Updates

### Before

The form was not present on Add Item view. List View showed all items in the database.

### After
Form present and adds items to the database with the user's token. The list view shows Items in the database associated with the user's token.



![Screenshot 2022-04-15 150053](https://user-images.githubusercontent.com/38439541/163622518-4cd8d50f-b96c-4819-a46b-0fbb0fff711b.png)
![Screenshot 2022-04-15 150317](https://user-images.githubusercontent.com/38439541/163622529-26d3efe7-4812-47f8-94ae-c945a9f52819.png)

## Testing Steps / QA Criteria



1. From your terminal, pull down this branch with git pull origin jc-cf-new-item-form and check that branch out with git checkout jc-cf-new-item-form.
2. Then use npm install to install dependencies locally and npm start to launch the app.
3. Click "create a new list" button.
4. Click "add an item" link at the bottom of the page.
5. Use form to add an item(s) to the list.
6. Click "list of items" link at the bottom of the page and check that your new item(s) is added.
